### ISSUE TITLE
Unable to run digitizeRadii() from command line

### ISSUE THREAD

**Issue by adospace** [droglenc/RFishBC#45]
**Describe the bug**
Using RScript.exe from the command line it's not possible to run digitizeRadii() 

**To Reproduce**
Create an R file with this content:
```
setwd("C:\\Users\\adosp\\AppData\\Roaming\\IowaDnr.DesktopApp\\tmpBDCF")
library(RFishBC)
library(dplyr)
library(RJSONIO)
digitizeRadii("Scale_1.jpg",id="1",reading="HBO",edgeIsAnnulus=FALSE)
```

than run it with the command:
C:\Program Files\R\R-4.0.3\bin\x64> Rscript.exe "path-to-R-file"

it returns:
Error in eval(predvars, data, env) : object "y" not found
Calls: digitizeRadii ... lines.formula -> eval -> eval -> <Anonymous> -> eval -> eval

**Expected behavior**
It should open the window with the image as it does in RGui

**Desktop (please complete the following information):**
 - OS: Windows
 - Version latest

**Additional context**
It may be somenthing with the default device that from the command line is "pdf" and not "windows"?

Thanks in advance for any hint!

**Comment by keithhurley** [comment]
did you ever find a solution to this?

**Comment by adospace** [comment]
No, I didn't. I somewhat solved running the rgui and sending a bounce of wm messages to it in order to open the image to digitalize

**Comment by keithhurley** [comment]
Thanks, I've been beating my head against this for two days.... I can do it
from a script or batch file directly and from the command line, but never
from a windows.process call.  I am guessing that the process ends when r
opens a device in a separate process and doesn't wait for the digitizing.

Any chance you have any code examples of how you used the wm messages?

Keith

**Comment by droglenc** [comment]
Unfortunately, I also do not have a solution for this yet.
